packages/builder/src/builderStore/storeUtils.js

```diff
@@ -79,3 +80,8 @@ export const regenerateCssForCurrentScreen = state => {
   ])
   return state
 }
+
+export const generateNewIdsForComponent = c =>
```

Comment:
Why do we need to do this?

Reply:
when we copy/duplicate a component it needs to be assigned new Ids